src/Web/Services/BasketViewModelService.cs

```diff
@@ -38,10 +39,13 @@ public async Task<BasketViewModel> GetOrCreateBasketForUser(string userName)
 
         private async Task<BasketViewModel> CreateViewModelFromBasket(Basket basket)
         {
-            var viewModel = new BasketViewModel();
-            viewModel.Id = basket.Id;
-            viewModel.BuyerId = basket.BuyerId;
-            viewModel.Items = await GetBasketItems(basket.Items); ;
+            var viewModel = new BasketViewModel
+            {
+                Id = basket.Id,
+                BuyerId = basket.BuyerId,
+                Items = await GetBasketItems(basket.Items)
+            };
+            ;
```

Comment:
It looks like this was there before you, but could you remove this straggling semi-colon there?

Reply:
Of course :)